server/app.js

```diff
@@ -6,7 +6,7 @@ const express = require('express')
 const path = require('path')
 const flash = require('connect-flash')
 const session = require('express-session')
-const ConnectRedis = require('connect-redis')
+const RedisStore = require('connect-redis').default
```

Comment:
as this is js and uses `require`, you have to explicitly reference the default export